casper/src/main/scala/coop/rchain/casper/rholang/RuntimeManager.scala

```diff
@@ -39,25 +40,24 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.FiniteDuration
 
 trait RuntimeManager[F[_]] {
-  def captureResults(
-      startHash: StateHash,
-      deploy: Signed[DeployData]
-  ): F[Seq[Par]]
   def replayComputeState(startHash: StateHash)(
       terms: Seq[ProcessedDeploy],
       systemDeploys: Seq[ProcessedSystemDeploy],
       blockData: BlockData,
-      withCostAccounting: Boolean
+      withCostAccounting: Boolean,
+      rand: Blake2b512Random
```

Comment:
Maybe it makes sense to put `rand` immediately after deploys because it's connected how deploys are executed. On other similar methods also.